well-known-bots.json

```diff
@@ -5628,14 +5632,15 @@
   {
     "id": "amazon-crawler",
     "categories": [
+      "ai",
```

Comment:
Same comment re: AppleBot.

Reply:
Dark Visitors says that amazonbot is AI: https://darkvisitors.com/agents/amazonbot

Reply:
```suggestion
```